File: p04_functions/exercises/factorial_division.py
def factorial(num1, num2):
    first_factorial = num1
    second_factorial = num2
    for num in range(num1-1, 0, -1):
        first_factorial *= num
    for num in range(num2-1, 0, -1):
        second_factorial *= num
    return f"{first_factorial / second_factorial:.2f}"



integer1 = int(input())
integer2 = int(input())
print(factorial(integer1, integer2))

File: SoftUni-JS-Front-End/functionsAndStatementsExercise/factorialDivision.js
function solve(num1, num2) {
    const factorial = function (x, y) {
        if (y <= 1) {
            return x
        }
        return factorial((x * y), y-1)
    }
    return (factorial(num1, num1-1) / factorial(num2, num2-1)).toFixed(2)
}
console.log(solve(2, 1))